data/content/assets/glyph-pocket-archive-16.svg

```diff
@@ -0,0 +1 @@
+<svg width="16" height="16" xmlns="http://www.w3.org/2000/svg"><path d="M1 14.667V4h14v10.667c0 .736-.597 1.333-1.333 1.333H2.333A1.333 1.333 0 0 1 1 14.667zM1 0h14a1 1 0 0 1 0 2H1a1 1 0 1 1 0-2zm9.341 7.247l-3.295 2.884-.839-.838a1 1 0 1 0-1.414 1.414l1.5 1.5a1 1 0 0 0 1.366.046l4-3.5a1 1 0 0 0-1.318-1.506z" fill="#000" fill-rule="evenodd"/></svg>
```

Comment:
The rest of the svgs use context fill instead of hardcoded values, so this will lead to unexpected behaviors:
![Screen Shot 2019-04-03 at 11 56 03 AM](https://user-images.githubusercontent.com/438537/55505271-9c62ef00-5607-11e9-8fe2-b456f1498ca3.png)

Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1541543

Reply:
Good catch! Submitted PR with the fix https://github.com/mozilla/activity-stream/pull/4882